test/test_bulk_renewal.js

```diff
@@ -3,7 +3,8 @@ const HashRegistrar = artifacts.require('@ensdomains/ens/HashRegistrar');
 const PublicResolver = artifacts.require('@ensdomains/resolver/PublicResolver');
```

Comment:
Additionally please rename, so we are consistent with other test files. `TestBulkRenewal`. This must have fallen through the previous refactoring.